src/WinGet.RestSource/Models/Schemas/Information.cs

```diff
@@ -20,8 +21,21 @@ public class Information
         /// </summary>
         public Information()
         {
-            this.SourceIdentifier = ApiConstants.SourceIdentifier;
+            this.SourceIdentifier = string.IsNullOrEmpty(ApiConstants.SourceIdentifier) ? "DefaultSourceIdentifier" : ApiConstants.SourceIdentifier;
             this.ServerSupportedVersions = ApiConstants.ServerSupportedVersions;
+
+            // Example of how Unsupported Package match fields can be used.
+            this.UnsupportedPackageMatchFields = ApiConstants.UnSupportedPackageMatchFields;
+            this.RequiredPackageMatchFields = ApiConstants.RequiredPackageMatchFields;

```

Comment:
Its unclear that we would add unsupported fields to the stock reference implementation. The documentation Roy has been putting together is working towards how one would quickly and easily take the reference implementation and leverage it in their business.

Usage examples should be in documentation, not in the default implementation.

Reply:
Removed the example